pkgs/development/tools/misc/go-licenses/default.nix

```diff
@@ -26,13 +27,21 @@ buildGoModule rec {
 
   nativeBuildInputs = [
     installShellFiles
+    makeWrapper
   ];
 
+  subPackages = [ "." ];
+
   postInstall = ''
     installShellCompletion --cmd go-licenses \
       --bash <("$out/bin/go-licenses" completion bash) \
       --fish <("$out/bin/go-licenses" completion fish) \
       --zsh  <("$out/bin/go-licenses" completion zsh)
+
+    # workaround empty output when GOROOT differs from built environment
+    # see https://github.com/google/go-licenses/issues/149
+    wrapProgram $out/bin/go-licenses \
+      --run 'export GOROOT=$(dirname $(realpath $(which go)))/..'
```

Comment:
Why depend on Go and which being in the PATH? Would it not be better to add them as a dependency?

Reply:
The program itself has no dependency on it and triggering a rebuild just to change the PATH in the wrapper is wasteful but it would simplify the wrapper